app/planets/model/mongo.go

```diff
@@ -0,0 +1,38 @@
+package model
+
+import "time"
+
+const (
+	dayInHours = 24
+)
+
+type PlanetMongo struct {
+	ID               string `bson:"_id"`
+	Name             string
+	Climate          []string
+	Terrain          []string
+	MovieAppearances int
+	CreationDate     time.Time
+}
+
+func (m PlanetMongo) ToDomain() Planet {
+	difference := time.Now().Sub(m.CreationDate)
+	cacheInDays := difference.Hours() / dayInHours
+	return Planet{
+		ID:               m.ID,
+		Name:             m.Name,
+		Climate:          m.Climate,
+		Terrain:          m.Terrain,
+		MovieAppearances: m.MovieAppearances,
+		CacheInDays:      int(cacheInDays),
+	}
+}
+
+func ToDomainList(mongoPlanets []PlanetMongo) []Planet {
+	var domainList []Planet
```

Comment:
Dica de otimização: quando você sabe o tamanho final do slice, já cria ele com esse tamanho, dependendo da quantidade de dados, você salva bastante cpu/memória.

Nesse caso aqui ficaria `domainList := make([]Planet, 0, len(mongoPlanets))`

Escrevi um [post](https://blog.eduardobonzi.dev/improving-performance-with-slices-in-go) sobre isso se quiser dar uma lida ;)